### PR TITLE
Fixed Mandragora Wardens in The Boyahda Tree and Yoran-Oran

### PR DESCRIPTION
**Pull request by kaincenteno**
npc_list.sql:
Renamed 2 Mandragora_Warden to Mandragora_Warden_Left and
Mandragora_Warden_Right as they offer different CS. The third one was
left as is.

Mandragora_Warden.lua:
Added CS if player is holding item.
Added standard CS (as currently npc only spoke if on mission)

Mandragora_Warden_Left.lua and Mandragora_Warden_Right.lua:
Files not existing, created new ones.
Added standard CS (they do not have other ones)

Yoran-Oran.lua:
Removed ending semicolons.
No Logic was changed.
Added Parameter for CS 439 as they were missing
Fixed Parameter for CS 470 as he was showing blank text for it.

### CODE REVIEW

File: scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua [hunk 1]
```diff
@@ -21,8 +21,10 @@ end
 function onTrigger(player,npc)
     if player:getVar("MissionStatus") == 4 then
         local dialog = player:getVar("mandialog")
-
-        if dialog == 0 then
+        if player:hasItem(1181) then
+            player:startEvent(10)
+            player:PrintToPlayer("The warden appears to want something from you...")
```
Comment (KnowOne134) [inline scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua]: Print to player? If its supose to day something it will be an event or a textid

Comment (kaincenteno) [reply scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua]: Hi @KnowOne134 😸

I know, I just couldn't find the textid for any of the PrintToPlayer that already existed for this in:
 
https://github.com/KnowOne134/FFXI_Events/blob/master/the_boyahda_tree.json
😿 not even for the one that I added. (Can be seen in https://www.youtube.com/watch?v=L1TGPHzvwzg)

Comment (TeoTwawki) [reply scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua]: Could you use a timestamped link please? it is a very long video to be scrolling around in.

It is likely a cs event and possibly requires specific parameters passed to said cs event. in any case, printToPlayer to fake it is not something we do in the project. That function is mainly for debugging and for people to make their custom things out of.

Comment (ibm2431) [reply scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua]:
```
  <thing type="DialogTableEntry">
    <field name="index">7333</field>
    <field name="text">≺Possible Special Code: 1F≻≺Possible Special Code: 0F≻The warden appears to want something from you...≺Prompt≻</field>
  </thing>
```

Comment (kaincenteno) [reply scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua]: Hi @KnowOne134 :

Changed it to use messageText and messageSpecial, and added the text to IDs.lua

@TeoTwawki :

This is a video capture from @Wiggo32 😸  https://youtu.be/7GekeM7HOUU?t=2513

@ibm2431  thank you for the info 😺

I am always thankful for your help. Can't wait for the mandragora's to be lively in oldschool server

**Comment by kaincenteno** [comment]
@KnowOne134  made the changes 😸 now yoran-oran is using npc util.

File: scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua [hunk 2]
```diff
@@ -19,21 +19,11 @@ function onTrade(player,npc,trade)
...
-            player:PrintToPlayer("Seems like he wants something")
-        elseif dialog == 1 then
-            player:startEvent(11)
-            player:setVar("mandialog", 2)
-        elseif dialog == 2 then
-            player:startEvent(12)
-            player:setVar("mandialog", 3)
-            player:PrintToPlayer("Seems like he wants some Gobbu Hummus")
-        end
+    if player:getVar("MissionStatus") == 4  or player:getVar("MissionStatus") == 5 then
```
Comment (TeoTwawki) [inline scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua]: Shouldn't that check which mission it is to? I mean lots of missions have a status 4 or status 5 at some point right?

**Pull request merged**